### PR TITLE
Plaintext export api

### PR DESCRIPTION
**Pull request by nikolauska**
Added functionality for api to export board in human readable format

### CODE REVIEW

**Comment by n4sjamk-docker** [comment]
Can one of the admins verify this patch?

File: utils/export.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+				'------------------------------------------\n';
+	}).join('') + '\n' +
+	'=========================================';
+}
+
+/**
+ * Generates CSV from board and tickets
+ * @param {object} board - Board object to be generated.
+ * @param {object} tickets - Tickets to be generated.
+ */
```
Comment (tanelih) [inline utils/export.js]: Add JSDoc `@return(s)` parameters

File: nohup.out [hunk 2]
```diff
@@ -0,0 +1,24 @@
+
```
Comment (tanelih) [inline nohup.out]: wtf

File: routes/board.js [hunk 3]
```diff
@@ -15,6 +15,8 @@ var Ticket = mongoose.model('ticket');
 var Router   = express.Router();
 var ObjectId = mongoose.Types.ObjectId;
 
+var exportFunctions = require('../utils/export');
```
Comment (tanelih) [inline routes/board.js]: Maybe consider renaming `exportFunctions` to something like `exportAs` ... ???

**Comment by tanelih** [comment]
Looks fine.

**Pull request merged**